behat/tests/features-v2/reporting/sections/visits-care/visits-care.feature

```diff
@@ -0,0 +1,66 @@
+@v2 @visits-care
+Feature: Visits and Care
+
+  Scenario: A user lives with a client that doesn't receive paid care and doesn't have a care plan.
+    Given a Lay Deputy has not started a report
+    And I view and start the visits and care report section
+    Then I should be on the live with client page
+    When I choose yes and save on the live with the client section
+    Then I should be on the client receive paid care page
+    When I choose no and save on the client receive paid care section
+    Then I should be on the who is doing the caring page
+    When I fill out and save the who is doing caring section
+    Then I should be on the does the client have a care plan page
+    When I choose no and save on the client has care plan section
+    Then I should be on the visits and care report summary page
+    And I should see the expected visits and care report section responses
+    And I should see all the additional information I gave for visit and care
+    And the previous section should be "Contacts"
+    And the next section should be "Bank accounts"
```

Comment:
To clarify here, yeah we decided this was so generic for the sections that it might be best just to have one navigate sections feature that goes over a completed report and checks all the sections so we don't accidentally miss any. Keep the features for checking buttons/links within the section itself :)